brainzutils/musicbrainz_db/tests/test_helper.py

```diff
@@ -1,80 +1,79 @@
 from collections import defaultdict
-from unittest import TestCase
-from unittest.mock import MagicMock
+
+import pytest
 from mbdata import models
+
 from brainzutils.musicbrainz_db.serialize import serialize_relationships
 from brainzutils.musicbrainz_db.helpers import get_relationship_info
 import brainzutils.musicbrainz_db as mb
 from brainzutils.musicbrainz_db.helpers import get_tags
-from brainzutils.musicbrainz_db.test_data import linkplaceurl_1, linkplaceurl_2, place_suisto
+from brainzutils.musicbrainz_db.utils import get_entities_by_gids
 
 
-class HelpersTestCase(TestCase):
+@pytest.mark.database
+class TestHelpers:
```

Comment:
These tests were using mocks earlier and began to fail after I updated _relationship_link_helper. These mocks look fragile anyway so I updated these to use real data.